File: 1/1-b.js
const readFile = require("fs").readFileSync;
const file = readFile("input", "utf-8").split("\n")

const readings = file.map( _ => parseInt(_.trim()));

const count = readings
    .map((v, i, arr) => v + arr[i + 1] + arr[i + 2])
    .reduce((acc, cur, i, arr) => acc += cur > arr[i - 1] ? 1 : 0, 0);

console.log(count)

File: 1/1-b.py
file = open("./input")

readings = list(map(lambda _: int(_), file.read().split("\n")))

previous = readings[0] + readings[1] + readings[2]
count = 0

for i in range(1, len(readings) - 2):
    reading = readings[i] + readings[i + 1] + readings[i + 2]
    if reading > previous: count += 1
    previous = reading

print(count)
